test/parallel/test-tls-keylog-tlsv13.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+
+const common = require('../common');
+if (!common.hasCrypto)
+  common.skip('missing crypto');
+
+const assert = require('assert');
+const events = require('events');
+const tls = require('tls');
+const fixtures = require('../common/fixtures');
+
+async function main() {
+  // Create server / client pair
+  const server = tls.createServer({
+    key: fixtures.readSync('/keys/agent2-key.pem'),
+    cert: fixtures.readSync('/keys/agent2-cert.pem'),
+    minVersion: 'TLSv1.3',
```

Comment:
does this feature work only with TLSv1.3? The docs didn't say anything to indicate that, nor does https://www.openssl.org/docs/man1.1.1/man3/SSL_CTX_set_keylog_callback.html.

Reply:
Logging is different on TLSv1.3 connections (which generate 5 secrets in total, at various stages of the connection), than on <=TLSv1.2 connections, for which a single `CLIENT_RANDOM` secret is generated. If you want, I can make another test for <=TLSv1.2.

Reply:
More info: https://security.stackexchange.com/a/42350/5067

Reply:
Makes sense. I don't think we have to test every possible protocol permutation without reason. Perhaps a single comment with the info above would be good in the test source.

Reply:
Added!